addons/addon-base-raas/packages/base-raas-services/lib/environment/__tests__/environment-mount-service.test.js

```diff
@@ -45,9 +45,9 @@ describe('EnvironmentMountService', () => {
   let service = null;
   let environmentScService = null;
   let iamService = null;
-  let studyService = null;
-  let studyPermissionService = null;
-  let aws = null;
+  // let studyService = null;
```

Comment:
Nit: Remove the services if they're not needed anymore?

Reply:
Done